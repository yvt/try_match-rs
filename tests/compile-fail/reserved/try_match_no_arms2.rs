fn main() {
    let _ = try_match::try_match!(Some(42));
}
