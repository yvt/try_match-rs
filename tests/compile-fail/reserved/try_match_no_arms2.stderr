error: unexpected end of macro invocation
 --> tests/compile-fail/reserved/try_match_no_arms2.rs:2:43
  |
2 |     let _ = try_match::try_match!(Some(42));
  |                                           ^ missing tokens in macro arguments
  |
note: while trying to match `,`
 --> src/lib.rs
  |
  |     ($in:expr, $p:pat $(if $guard:expr)? => $out:expr $(,)?) => {
  |              ^
